MAUI/Picker/getting-started.md

```diff
@@ -0,0 +1,308 @@
+
+## Set header to the Picker
+
+The SfPicker control allows you to add the header text by setting the [Text] property in the [PickerHeaderView]. To enable the header view by setting the [Height] property in [PickerHeaderView].
+
+{% tabs %}
+{% highlight xaml tabtitle="XAML" hl_lines="3" %}
+
+<picker:SfPicker x:Name="picker">
+    <picker:SfPicker.HeaderView>
+        <picker:PickerHeaderView Text="Select a color" Height="40" />
+    </picker:SfPicker.HeaderView>
+</picker:SfPicker>
+
+{% endhighlight %}
+{% highlight c# tabtitle="C#" hl_lines="4 5" %}
+
+SfPicker picker = new SfPicker();
+picker.HeaderView = new PickerHeaderView()
+{
+    Text = "Select a color",
+    Height = 40,
+};
+
+this.Content = picker;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Add picker items to the picker
+
+The .NET MAUI Picker control is a data bounded control. Hence, you must create collection of data’s and bind it to the picker control.
+
+* Create a simple Observable Collection of data for the picker,
+
+{% tabs %}
+{% highlight c# tabtitle="ItemInfo.cs" %}
+
+public class ItemInfo
+{
+    private ObservableCollection<object> dataSource = new ObservableCollection<object>()
+    {
+        "Pink", "Green", "Blue", "Yellow", "Orange", "Purple", "SkyBlue", "PaleGreen"
+    };
+
+    public ObservableCollection<object> DataSource
+    {
+        get
+        {
+            return dataSource;
+        }
+        set
+        {
+            dataSource = value;
+        }
+    }
+
+    public ItemInfo()
+    {
+
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+* Bind the Collection to picker
+
+Picker control allows you to bind the collection of data by setting the [ItemsSource] property from [PickerColumn]. You can bind the collection of data in both XAML or C#.
+
+{% tabs %}
+{% highlight xaml tabtitle="XAML" hl_lines="6 10 11 12" %}
+
+<picker:SfPicker x:Name="picker">
+    <picker:SfPicker.HeaderView>
+        <picker:PickerHeaderView Text="Select a color" Height="40" />
+    </picker:SfPicker.HeaderView>
+    <picker:SfPicker.Columns>
+        <picker:PickerColumn ItemsSource="{Binding DataSource}" />
+    </picker:SfPicker.Columns>
+</picker:SfPicker>
+
+<ContentPage.BindingContext>
+    <local:ItemInfo />
+</ContentPage.BindingContext>
+
+{% endhighlight %}
+{% highlight c# tabtitle="C#" hl_lines="1 8" %}
+
+ItemInfo itemInfo = new ItemInfo();
+SfPicker picker = new SfPicker()
+{
+    Columns = new ObservableCollection<PickerColumn>()
+    {
+        new PickerColumn()
+        {
+            ItemsSource = itemInfo.DataSource,
+        }
+    }
+};
+
+this.Content = picker;
+
+{% endhighlight %}
+{% endtabs %}
+
+
+## Set footer to the Picker
+
+In SfPicker control, validation buttons (OK and Cancel) can be customized by setting the [OkButtonText] and [CancelButtonText] property in the [PickerFooterView]. It allows you to confirm or cancel the selected item.  The `OkButtonText` can be enabled by using the [ShowOkButton] property in [PickerFooterView].
```

Comment:
In SfPicker control, validation buttons (OK and Cancel) can be customized by setting the [OkButtonText] and [CancelButtonText] properties in the [PickerFooterView]. It allows you to confirm or cancel the selected item.  The `OkButtonText` can be enabled using the [ShowOkButton] property in the [PickerFooterView].

Reply:
Content modified.